### DISCUSSION TITLE
Would it be possible only to get one notification e-mail if there is an extension update?

### DISCUSSION THREAD

**Discussion by drdingens** [akeeba/panopticon#781] [Q&A (Community Support)] [answered]
Hey
I am getting an email every thirty minutes if there is an extension update on my wordpress sites. Would it be possible to receive this e-mail only once?
Thanks
Rolf

**Comment by nikosdion** [comment, marked as answer]
https://github.com/akeeba/panopticon/commit/30ecee9f39dda9bcd0a036214a9df8b0f281e788

**Reply by drdingens** [reply]
Thanks, do I need to do anything?
Cheers
Rolf

**Reply by nikosdion** [reply]
You should wait for the next release which will be in 1-2 weeks.

**Reply by drdingens** [reply]
Okay, thanks!
Rolf
